babel.config.json

```diff
@@ -4,8 +4,14 @@
       "bugfixes": true,
       "shippedProposals": true,
       "corejs": 3,
-      "useBuiltIns": "entry"
-    }]
+      "useBuiltIns": "entry",
+      "include": [
+        "@babel/proposal-nullish-coalescing-operator",
+        "@babel/proposal-optional-chaining"
+      ]
+    }],
+    ["@babel/react", { "useSpread": true }],
+    "@babel/typescript"
   ],
   "plugins": [
     ["@babel/transform-runtime", { "useESModules": true }],
```

Comment:
We might have to add `@babel/runtime` in our `dependencies` in the future, but currently no helpers get imported in our bundles.